src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -1295,6 +1298,25 @@ void VM_Version::get_processor_features() {
     FLAG_SET_DEFAULT(MaxVectorSize, max_vector_size);
   }
 
+  if (FLAG_IS_DEFAULT(SuperWordMaxVectorSize)) {
+    if (FLAG_IS_DEFAULT(UseAVX) && UseAVX > 2 &&
+        is_intel_skylake() && _stepping > 5) {
```

Comment:
Should you check `_stepping >= 5`?  Otherwise `_stepping == 5` is missing in all adjustments.